src/inscribeit/swap_inscription.cpp

```diff
@@ -788,6 +823,14 @@ SwapInscriptionBuilder &SwapInscriptionBuilder::OrdUTXO(const string &txid, uint
     return *this;
 }
 
+CAmount SwapInscriptionBuilder::getWholeFee(CAmount fee_rate) {
```

Comment:
OrdCommitTx has an individual fee_rate. SwapInscriptionBuilder::getWholeFee() should take it into an account.